p4broker/hooks/init

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -xe
+#
+
+if [ ! -d {{pkg.svc_data_path}}/log ]; then
+	mkdir -p {{pkg.svc_data_path}}/log
+	chown -R hab:hab {{pkg.svc_data_path}}/log
+	rm -rf {{pkg.svc_data_path}}/log/p4broker.log
```

Comment:
Is this even possible? The check above verified that the directory didn't exist so this file should also not exist?

Reply:
Whoops, copied too much code.